pod.go

```diff
@@ -547,11 +547,6 @@ func createPod(podConfig PodConfig) (*Pod, error) {
 		return nil, err
 	}
 
-	// Passthrough devices
-	if err := p.attachDevices(); err != nil {
-		return nil, err
-	}
-
```

Comment:
@devimc I am not sure what broke, but I think this may not work with cold-plugging vfio devices.
In any case we need to move towards hot-plugging vfio devices as well, now that hot-plug issues with vfio have been resolved.